### PR TITLE
Remove browser autocomplete in new file menu

### PR DESCRIPTION
**Pull request by PVince81**
That got annoying after a while...

Please review @jancborchardt @LukasReschke @icewind1991 @rullzer @MorrisJobke

### CODE REVIEW

**Comment by rullzer** [comment]
:+1:

**Comment by LukasReschke** [comment]
Works :+1:

**Comment by MorrisJobke** [comment]
@karlitschek Is a backport to stable9 fine?

**Comment by karlitschek** [comment]
backport is fine :+1:

**Comment by jancborchardt** [comment]
Yeah, good call. Also on backport.

**Comment by PVince81** [comment]
I think 8.2 has it too, because that's where the problem appeared.
So, backport to stable8.2 and stable9

**Comment by MorrisJobke** [comment]
> So, backport to stable8.2 and stable9

Could you create the backport PRs?

**Comment by PVince81** [comment]
Will do

**Comment by PVince81** [comment]
stable9: https://github.com/owncloud/core/pull/22926
stable8.2: https://github.com/owncloud/core/pull/22927

File: apps/files/js/newfilemenu.js [hunk 1]
```diff
@@ -27,7 +27,7 @@
 	var TEMPLATE_FILENAME_FORM =
 		'<form class="filenameform">' +
 		'<label class="hidden-visually" for="{{cid}}-input-{{fileType}}">{{fileName}}</label>' +
-		'<input id="{{cid}}-input-{{fileType}}" type="text" value="{{fileName}}">' +
+		'<input id="{{cid}}-input-{{fileType}}" type="text" value="{{fileName}}" autocomplete="off" autocapitalize="off" autocorrect="off">' +
```
Comment (MorrisJobke) [inline apps/files/js/newfilemenu.js]: Why the autocorrect statement? Often I want to use the autocorrection also for filenames. Only the autocomplete and autocapitalize should be fine.

Comment (MorrisJobke) [inline apps/files/js/newfilemenu.js]: https://github.com/owncloud/core/pull/22986

**Pull request merged**